client-sdk/go/helpers/address.go

```diff
@@ -12,15 +12,18 @@ import (
 	"github.com/oasisprotocol/oasis-sdk/client-sdk/go/config"
 	"github.com/oasisprotocol/oasis-sdk/client-sdk/go/crypto/signature/secp256k1"
 	"github.com/oasisprotocol/oasis-sdk/client-sdk/go/modules/rewards"
+	"github.com/oasisprotocol/oasis-sdk/client-sdk/go/testing"
 	"github.com/oasisprotocol/oasis-sdk/client-sdk/go/types"
 )
 
 const (
-	addressPrefixOasis       = "oasis1"
-	addressPrefixEth         = "0x"
-	addressExplicitSeparator = ":"
+	addressPrefixOasis = "oasis1"
+	addressPrefixEth   = "0x"
+
+	AddressExplicitSeparator = ":"
```

Comment:
These should all be private and a separate parse method should be exported as mentioned above.